index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Direct Sockets API
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"
+    defer></script>
+    <script class='remove'>
+      var respecConfig = {
+        specStatus: "unofficial",
+        github: {
+          repoURL: "WICG/raw-sockets",
+          branch: "main"
+        },
+        group: "wicg",
+        editors: [
+        {
+          name: "Eric Willigers",
+          company: "Google Inc.",
+          companyURL: "https://google.com",
+          w3cid: 67534
+        }],
+        xref: "web-platform",
+        // eventually add mdn: true, and caniuse: "direct-sockets"
+      };
+    </script>
+  </head>
+  <body data-cite="Direct Sockets">
+    <section id="abstract">
+      <p>
+        This specification defines an API that allows web applications to talk
+        to servers and devices that have their own protocols incompatible with
+        what’s available on the web.
```

Comment:
super-nit: tone of "what's" is informal. Perhaps replace with "protocols" or "those".

Reply:
those